source/UI.cpp

```diff
@@ -32,6 +32,17 @@ UI::UI()
 
 
 
+bool UI::AllowFastForward() const
+{
+	int count=0;
+	for(const auto &it : stack)
+		if(!it->AllowFastForward())
+			count++;
```

Comment:
Any non-zero count returns in returning false, why not just return false immediately instead of incrementing count?

Reply:
This was leftover from debugging. I've removed it.